jiant/trainer.py

```diff
@@ -436,6 +435,9 @@ def get_sampling_weights(
             sample_weights = 1 / task_n_train_examples
         elif weighting_method == "inverse_log_example":
             sample_weights = 1 / np.log(task_n_train_examples)
+        elif "examples-proportional-mixing" in weighting_method:
+            K = int(weighting_method.split("K=")[1])
```

Comment:
At least under GitHub's syntax highlighting, it seems like 'K' is being treated as if it were reserved...

Reply:
Update defaults.conf to document this option.